media/js/base/site.js

```diff
@@ -146,12 +146,17 @@
         // to avoid lots of flickering
         var platform = window.site.platform = window.site.getPlatform();
         var version = window.site.platformVersion = window.site.getPlatformVersion();
+        var _version = version ? parseFloat(version) : 0;
 
         if (platform === 'windows') {
             // Add class to support downloading Firefox for Windows 64-bit on Windows 7 and later
-            if (version && parseFloat(version) >= 6.1) {
+            if (_version >= 6.1) {
                 h.className += ' win7up';
             }
+        } else if (platform === 'oldwin' && _version >= 5.1 && _version <= 6) {
+            // Add class to allow Windows XP/Vista users to download Firefox 52 ESR, though these
+            // legacy platforms are now officially unsupported
+            h.className += ' xpvista';
```

Comment:
This is confusing since the system class name ends up as `windows xpvista`, yet `window.site.platform` returns `oldwin`